.github/workflows/build.yml

```diff
@@ -0,0 +1,100 @@
+---
+name: Create packages and test installation
+
+on:
+  pull_request:
+
+jobs:
+  # XXX done outside of the build matrix due to different container name
+  build-centos7:
+    name: Build CentOS 7 RPMs
+    runs-on: ubuntu-latest
+    container: quay.io/centos/centos:7
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Install build requisites
+        run: |
+          yum install -y rpm-build yum-utils rpmlint
+          yum-builddep -y bdii-config-site.spec
+      - name: build rpm
+        run: |
+          make clean rpm
+          rpmlint --file .rpmlint.ini build/RPMS/noarch/*.el7.noarch.rpm
+      - name: Upload RPMs
+        uses: actions/upload-artifact@v3
+        with:
+          name: rpms7
+          path: |
+            build/RPMS/noarch/bdii-config-site-*.el7.noarch.rpm
+
+  # Use a matrix for CentOS Stream versions
+  build:
+    strategy:
+      matrix:
+        centos-version: [8, 9]
+    name: Build CentOS ${{ matrix.centos-version }} RPMs
+    runs-on: ubuntu-latest
+    container: quay.io/centos/centos:stream${{ matrix.centos-version }}
```

Comment:
maybe we should use almalinux or rockylinux instead ? i think nothing will break but those are the OS to be supported now

Reply:
Until now I just (lazily...) followed with what I did for other packages, building on CentOS Stream, especially as we don't expect much impact/differences especially for the python / shell stuff.
But I agree switching is probably best. I was thinking of switching to Alma at some point, as it seems it's expected to be the one that will be selected / "official" in the end.
So let's try to start building on Alma this package.

Reply:
Will try using their official containers: https://wiki.almalinux.org/containers/docker-images.html 

Will do it for both 8 and 9. Will be simpler than keeping something on CentOS Stream for 8 and only move Alma to 8, and I guess it should be OK.

Reply:
thanks!